game-core/src/main/java/games/strategy/sound/ClipPlayer.java

```diff
@@ -124,13 +124,12 @@ static synchronized ClipPlayer getInstance() {
     return clipPlayer;
   }
 
-  public static synchronized ClipPlayer getInstance(final ResourceLoader resourceLoader) {
+  public static synchronized void getInstance(final ResourceLoader resourceLoader) {
```

Comment:
Possibly outside the scope of this PR, but the method name no longer makes sense.  Would suggest changing it to something like `initialize()` or anything that conveys the notion of creating the `ClipPlayer` singleton if necessary.

Reply:
Good suggestion, though will leave this one out of scope.